policyengine_us/parameters/gov/contrib/states/ri/ctc/phaseout/earnings_based/in_effect.yaml

```diff
@@ -0,0 +1,9 @@
+description: The Rhode Island CTC uses earnings-based phaseout if this is true.
```

Comment:
```suggestion
description: Rhode Island phases the state child tax credit out based on household earnings, if this is true.
```

Reply:
Same for other descriptions: spell out acronyms such as CTC and AGI and be description